### ISSUE TITLE
label-content-name-mismatch should ignore SVG titles

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4509]
This shouldn't fail the label-content-name-mismatch rule:

```html
<button aria-label="Options">
  Options
  <svg xmlns="http://www.w3.org/2000/svg" width="15" height="10" viewBox="0 0 15 10" fill="none">
    <title>Caret</title>
    <path d="M13.4862 0.756836L14.9004 2.17104L7.82935 9.24214L0.75825 2.17104L2.17245 0.756835L7.82935 6.41374L13.4862 0.756836Z" fill="black"></path>
  </svg>
</button>
```

**Comment by WilcoFiers** [comment]
Actually, turns out we already have an issue open for this: https://github.com/dequelabs/axe-core/issues/4065
